zeek

```diff
@@ -89,15 +63,58 @@ start)
 		exit 0
 	fi
 
+	# create bare minimum directories required for running Zeek
+	if [ ! -d "$HOST_ZEEK/logs" ] || [ ! -d "$HOST_ZEEK/spool" ] || [ ! -d "$HOST_ZEEK/etc" ]; then
+		sudo mkdir -p "$HOST_ZEEK/logs" "$HOST_ZEEK/spool" "$HOST_ZEEK/etc"
+	fi
+
+	# create the node.cfg file required for running Zeek
+	if [ ! -s "$HOST_ZEEK/etc/node.cfg" ]; then
+		echo "Could not find $HOST_ZEEK/etc/node.cfg. Generating one now." >&2
+
+		$SUDO docker run --rm -it --network host \
+		--mount source="$HOST_ZEEK/etc/",destination=/zeek/,type=bind \
+		"$IMAGE_NAME" \
+		zeekcfg -o /zeek/node.cfg --type afpacket
+	fi
+
+	# create the volumes required for peristing user-installed zkg packages
+	$SUDO docker volume create zeek-zkg-script >/dev/null
+	$SUDO docker volume create zeek-zkg-plugin >/dev/null
+	$SUDO docker volume create zeek-zkg-state >/dev/null
```

Comment:
Took me forever to track down everything required for `zkg install` packages to persist restarts.